packages/rich-text-links/src/index.ts

```diff
@@ -49,7 +50,7 @@ export function getRichTextEntityLinks(
   /**
    *  An instance of a Rich Text Document.
    */
-  document: Document,
+  document: Document | null,
```

Comment:
I would refrain from weakening this very public function signature.

Reply:
I agree, but I will open a ticket for DALI to fix `@contentful/models` to make sure they stop sending `null` as a parameter value to prevent this type issue breaking production again in the future